cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -462,8 +463,13 @@ def test_custom_button_role_access_service(context):
     pass
 
 
-@pytest.mark.manual
-def test_custom_button_dialog_service_archived():
+@test_requirements.customer_stories
+@pytest.mark.meta(automates=[BZ(1439883)])
+@pytest.mark.provider([VMwareProvider], override=True, scope="module")
```

Comment:
Why `module` scope here? If it is module scope, why not put it at the top of the file in `pytestmark`?

Reply:
Actually most of test in this file using `generic service` no need of provider that why I avoid mark provider globally. The second why module yup no need of it even function scope work but it will not effect. can i move to fictional scope?

Reply:
Yeah kind of a minor point but I would leave it at `function` scope.